babs/cli.py

```diff
@@ -887,14 +983,14 @@ def get_existing_babs_proj(project_root):
     """
 
     # Sanity check: the path `project_root` exists:
-    if op.exists(project_root) is False:
+    if os.path.exists(project_root) is False:
         raise Exception(
             '`--project-root` does not exist! Requested `--project-root` was: ' + project_root
         )
 
     # Read configurations of BABS project from saved yaml file:
-    babs_proj_config_yaml = op.join(project_root, 'analysis/code/babs_proj_config.yaml')
-    if op.exists(babs_proj_config_yaml) is False:
+    babs_proj_config_yaml = os.path.join(project_root, 'analysis/code/babs_proj_config.yaml')
+    if os.path.exists(babs_proj_config_yaml) is False:
```

Comment:
does this not trigger a linter warning?

Reply:
For some reason it doesn't, but it's ugly so I fixed it.